app.js

```diff
@@ -26,6 +34,8 @@ if (isProduction) {
         "https://partner-platform.dev.opentargets.xyz",
         "https://platform.opentargets.org",
         "https://platform.dev.opentargets.xyz",
+        "https://localhost",
```

Comment:
is localhost supposed to be used in 'production'? I think this should be removed.

Reply:
This is running in node, right? I should be able to access environment variables passed to the Docker container. The server's hostname (as the user would see it in their browser address bar, usually a DNS name in production) is an environment variable we already pass to a number of other docker containers on startup.